packages/api/src/hooks/useCryptoTransactions.ts

```diff
@@ -20,11 +22,54 @@ type TModifiedTransaction = Omit<TTransaction, 'status_code' | 'transaction_type
         | { transaction_type: 'withdrawal'; status_code: TWithdrawalStatus }
     );
 
+const getFormattedConfirmations = (transaction: TModifiedTransaction) => {
+    switch (transaction.status_code) {
+        case 'CONFIRMED':
+            return 'Confirmed';
+        case 'ERROR':
+            return 'NA';
+        default:
+            return transaction.confirmations || 'Pending';
+    }
+};
+
+const getStatusName = (status_code: TModifiedTransaction['status_code']) => {
+    switch (status_code) {
+        case 'CONFIRMED':
+        case 'SENT':
+            return 'Successful';
+        case 'ERROR':
+        case 'REJECTED':
+        case 'REVERTED':
+            return 'Unsuccessful';
+        case 'PENDING':
+        case 'PERFORMING_BLOCKCHAIN_TXN':
+        case 'PROCESSING':
+        case 'REVERTING':
+        case 'VERIFIED':
+            return 'In process';
+        case 'CANCELLED':
+            return 'Cancelled';
+        case 'LOCKED':
+            return 'In review';
+        default:
+            return '';
```

Comment:
```suggestion
const getStatusName = (status_code: TModifiedTransaction['status_code']) => {
    switch (status_code) {
        case 'CONFIRMED':
        case 'SENT':
            return 'Successful';
        case 'ERROR':
        case 'REJECTED':
        case 'REVERTED':
            return 'Unsuccessful';
        case 'PENDING':
        case 'PERFORMING_BLOCKCHAIN_TXN':
        case 'PROCESSING':
        case 'REVERTING':
        case 'VERIFIED':
            return 'In process';
        case 'CANCELLED':
            return 'Cancelled';
        case 'LOCKED':
            return 'In review';
        default:
            return '';
```

hi @rostislav-deriv  i think these strings are already declared in `TransactionsPendingRow.tsx`. Can you create a constants file and update the ones used everywhere? Best to not duplicate things like string constants

Reply:
There's now a single source of truth in terms of these constants (in `useCryptoTransactions.ts`):

<img width="494" alt="Screenshot 2023-11-01 at 11 10 46" src="https://github.com/binary-com/deriv-app/assets/119863957/89a6749d-79f2-442f-a075-81d2205849af">